LayoutTests/TestExpectations

```diff
@@ -154,6 +154,9 @@ editing/pasteboard/paste-dark-mode-color-filtered.html [ Skip ]
 fast/forms/number/number-dark-appearance.html [ Skip ]
 fast/loader/plain-text-document-dark-mode.html [ Skip ]
 
+# Only applicable on platforms with vertical form control support
+fast/forms/vertical-writing-mode/progress.html [ Skip ]
```

Comment:
I'd rather blocklist platforms that don't have support rather than allowlist platforms that do.

Reply:
Done.